### DISCUSSION TITLE
meta_srv complete implementation of etcd？

### DISCUSSION THREAD

**Discussion by Euraxluo** [GreptimeTeam/greptimedb#2105] [Q&A] [answered]
I'm working on source code and architecture

**Comment by waynexia** [comment]
Are you asking if our `meta_srv` is implemented on top of etcd? If so, we are relying on etcd for a distributed k-v storage service

**Reply by Euraxluo** [reply]
Hello, I would like to further clarify the previous question. Regarding the meta_srv module, I want to confirm whether it relies on the existing etcd service for its implementation or if it has independently re-implemented similar functionalities to etcd? Thank you!

**Reply by MichaelScofield** [reply, marked as answer]
It relies on the existing etcd service.
